### ISSUE TITLE
[Files_videoviewer] Only some browsers can play videos.

### ISSUE THREAD

**Issue by SergioBertolinSG** [owncloud/apps#2065]
### Steps to reproduce

1. Upload a video.
2. Reproduce it.

### Expected behaviour
Video is reproduced.

### Actual behaviour
Chrome works.
Firefox works.
Safari doesn't work.
Opera doesn't work.
Internet Explorer 8 doesn't work.
Internet Explorer 9 works.

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
{"installed":true,"maintenance":false,"version":"8.1.0.5","versionstring":"8.1 beta 1","edition":"Enterprise"}

**Updated from an older ownCloud or fresh install:**
Fresh

**List of activated apps:**
Default in enterprise edition.
Encryption.

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
Yes

**Logs**
```
```

### Client configuration

**browser**
Chrome 41, Firefox 37, Opera 29, Internet Explorer 8, Safari 7

**Comment by RealRancor** [comment]
"Reproduce"?

Do you mean played?

**Comment by DeepDiver1975** [comment]
honestly not much we can do.

**Comment by libasys** [comment]
It is how a video file get streamed from the media player of a browser, safari and mobil ios safari streams the video file reading header "Content-Range" and you have to stream the video by bytes! 
http://codesamplez.com/programming/php-html5-video-streaming-tutorial
This should work with your browser:
https://ssl.webpack.de/demo81.libasyscloud.de/index.php/s/m23uB2dOMnzbtHP

**Comment by RealRancor** [comment]
@SergioBertolinSG Just had a short chat with a user on IRC. Seems he managed to get more videos to play by installing the VLC Video Player. This adds a browser plugin as well as additional codecs to the system. Worth a try...

**Comment by cmonteroluque** [comment]
sounds like a workaround

**Comment by svet-b** [comment]
I experienced a similar issue and was able to get streaming working by enabling X-Sendfile on the server side, following https://doc.owncloud.org/server/8.1/admin_manual/configuration_files/serving_static_files_configuration.html.

I posted my full diagnostics on https://forum.owncloud.org/viewtopic.php?f=31&t=29431. After spending a couple of days struggling with this, I feel that the relationship between streaming and X-Sendfile ought to be explicitly documented somewhere.
